organisations/tests/test_models.py

```diff
@@ -47,6 +47,59 @@ def test_membership_creation():
     assert Person.objects.count() == 1
 
 
+@pytest.mark.django_db
+def test_own_places():
+    place1 = "abc:123"
+    place2 = "xyz:321"
+    place3 = "jkl:999"
+    PersonFactory(name="person without places")
+    person1 = PersonFactory(name="person1", place_ids=[place1, place2])
+    person2 = PersonFactory(name="person2", place_ids=[place1])
+    person3 = PersonFactory(name="person3", place_ids=[place3, place2])
+    assert person1.place_ids == [place1, place2]
+    assert Person.objects.all().count() == 4
+    assert list(Person.objects.filter(place_ids__contains=[place1])) == [
+        person1,
+        person2,
+    ]
+    assert list(Person.objects.filter(place_ids__contains=[place2])) == [
+        person1,
+        person3,
+    ]
+    assert list(Person.objects.filter(place_ids__contains=[place1, place2])) == [
+        person1
+    ]
+    assert (
+        Person.objects.filter(place_ids__contains=[place1, place2, place3]).count() == 0
+    )
+
+    assert list(Person.objects.filter(place_ids__contained_by=[place1, place2])) == [
+        person1,
+        person2,
+    ]
+    assert list(
+        Person.objects.exclude(place_ids=[]).filter(
+            place_ids__contained_by=[place1, place2]
+        )
+    ) == [person1, person2]
+    assert (
+        Person.objects.filter(place_ids__contained_by=[place1, place2, place3]).count()
+        == 3
+    )
+
+    assert list(Person.objects.filter(place_ids__overlap=[place1])) == [
+        person1,
+        person2,
+    ]
+    assert list(Person.objects.filter(place_ids__overlap=[place1, place3])) == [
+        person1,
+        person2,
+        person3,
+    ]
+
+    assert list(Person.objects.filter(place_ids__len=1)) == [person2]
+
+
```

Comment:
I don't quite understand the meaning of this test, I mean isn't this basically just testing that Django's `ArrayField` works? 😄

Also, there is now no test to verify the GraphQL mutation works, I would rather do that 🙂

Reply:
first of all, I want it for myself! 😆  I know the field type is tested by Django already, but I really wanted to test its functionality, because I saw that it have had a bug in Django some time ago and it's a Postgre specific field. I originally wrote it while solving the problem with Api and speculating on whether the field needs to be forced to nullable.

So, if you don't mind, I leave it there. At least nobody should ever have to touch on it.

And yeah, I thought I missed something since this was a draft PR this morning. I'm going to add those mutation tests - and maybe even implement the feature first 😄